training/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/training
+spring.datasource.username=${POSTGRESUSERNAME}
+spring.datasource.password=${POSTGRESPASSWORD}
```

Comment:
Good Job Juan!!. That's a very good practice, we always recommend it :D

Reply:
Wonderful I appreciate it! :)